generators/generator-botbuilder/generators/app/templates/core/package-with-tests.json.js

```diff
@@ -17,10 +17,10 @@
     },
     "dependencies": {
         "@microsoft/recognizers-text-data-types-timex-expression": "1.1.4",
-        "botbuilder": "^4.23.1",
-        "botbuilder-ai": "^4.23.1",
-        "botbuilder-dialogs": "^4.23.1",
-        "botbuilder-testing": "^4.23.1",
+        "botbuilder": "4.1.6",

```

Comment:
any reason to do not use the latest?

Reply:
We wanted to keep the same version as the libraries, so it's clearer that the script will replace it.